app/views/pages/derivatives.html.erb

```diff
@@ -0,0 +1,160 @@
+  <p class="about_p"> The following guides walk through how to use AUK derivative files for further analysis.</p>
+  <div id="accordion">
+    <div class="accord">
+      <div class="accord_header" id="headingOne">
+        <h5 class="mb-0">
+          <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">Gephi & Raw Network Files
+          </button>
+        </h5>
+      </div>
+      <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordion">
+        <div class="accord_body">
+          <p class="about_p">When using the web as a historical source, the ability to see the way websites link to each other can be invaluable. However, using network analysis in historical research can also be a daunting prospect.</p>
+          <table id="accord_table">
+            <tbody class="tbody_table">
+              <tr class="tr_table">
+                <td class="td_table"><strong>Tutorial</strong></td>
+                <td class="td_table"><%=link_to("Network Graphing Archived Websites With Gephi", "/derivatives/gephi")%></td>
+              </tr>
+              <tr class="tr_table">
+                <td class="td_table"><strong>File Output</strong></td>
+                <td class="td_table"><kbd>.gexf</kbd> and <kbd>.graphml</kbd></td>
+              </tr>
+              <tr class="tr_table">
+                <td class="td_table"><strong>Files Generated</strong></td>
+                <td class="td_table">
+                  <ol>
+                    <li class= "about_li"><strong>Gephi file</strong>, which can be loaded into <a href="https://gephi.org/" target="_blank">Gephi</a>. It will have basic characteristics already computed and a basic layout.</li>
+                    <li class= "about_li"><strong>Raw Network</strong> file, which can also be loaded into <a href="https://gephi.org/" target="_blank">Gephi</a>. You will have to use that network program to lay it out yourself.</li>
+                  </ol>
+                </td>
+              </tr>
+            </tbody>
+          </table>   
+        </div>
+      </div>
+  </div>
+  <div class="accord">
+    <div class="accord_header" id="headingTwo">
+      <h5 class="mb-0">
+        <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">Plain Text Files
+        </button>
+      </h5>
+    </div>
+    <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+      <div class="accord_body">
+        <p class="about_p">Each website within the web archive collection will have its full text presented on one line, along with information around when it was crawled, the name of the domain, and the full URL of the content. AUK users will have access to the Full Text file which contains the extracted plain text of all the HTML files found within a collection.</p>
```

Comment:
Maybe change to:

The full text of a web collection can be invaluable for a researcher! AUK users will have access to the full text file. In it, each website within the web archive collection will have its full text presented on one line, along with information around when it was crawled, the name of the domain, and the full URL of the content.